src/test/java/net/masterthought/cucumber/json/support/TagObjectTest.java

```diff
@@ -0,0 +1,83 @@
+package net.masterthought.cucumber.json.support;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import net.masterthought.cucumber.ValidationException;
+
+/**
+ * @author Damian Szczepanik (damianszczepanik@github)
+ */
+public class TagObjectTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void TagObjectOnNullTagNameThrowsException() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2651355096/issues/source?bid=3059308&fileBranchId=3324685#l20)